pkg/util/netutil/netutil.go

```diff
@@ -39,10 +39,12 @@ func ActivateInterface(
 	gatewayIp string) (string, error) {
 
 	link, err := netlink.LinkByName(vethName)
-	if err == nil {
-		if link.Attrs().OperState == netlink.OperUp {
-			return fmt.Sprintf("Interface %s has already been UP.", vethName), nil
-		}
+	if err != nil {
+		return fmt.Sprintf("Failed to get Interface %s", vethName), err
+	}
+
+	if link.Attrs().OperState == netlink.OperUp {
+		return fmt.Sprintf("Interface %s has already been UP.", vethName), nil
 	}
```

Comment:
In line 137 below, do we need to check if netNSFileName is non-empty before executing ip netns del?

Reply:
We don't need. That part it's compose a command line script and will just print in the screen if there is error. Won't throw panic.